tests/framework/components/particlesystem/test_particlesystemcomponent.js

```diff
@@ -55,12 +55,13 @@ test("colorMapAsset removes events", function () {
     });
 });
 
-test("meshAsset removes events", function () {
+// TODO Skipped because box.json doesn't exist in the repo
```

Comment:
This file just seems to be missing – if we add it back in we can re-enable this test.

Reply:
My bad, I have this locally.